spec/helpers/application_helper_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+describe ApplicationHelper do
+  describe '.featured_collections' do
+    it 'should return the output of FeaturedItems.collections' do
+      expect(helper.featured_collections).to eq(FeaturedItems.collections)
+    end
+  end
+
+  describe '.featured_maps' do
+    it 'should return the output of FeaturedItems.maps' do
+      expect(helper.featured_maps).to eq(FeaturedItems.maps)
```

Comment:
same as before, test if the method is called alongside if the right array is returned